presto-main/src/main/java/com/facebook/presto/type/BigintOperators.java

```diff
@@ -111,7 +111,7 @@ public static long divide(@SqlType(StandardTypes.BIGINT) long left, @SqlType(Sta
             return left / right;
         }
         catch (ArithmeticException e) {
-            throw new PrestoException(DIVISION_BY_ZERO, e);
+            throw new PrestoException(DIVISION_BY_ZERO, "Division by zero");
```

Comment:
Personally I think stack trace is more helpful, and the error code name should be enough to convey the information of the added message

Reply:
@feilong-liu Makes sense, only thing is that when tests fail the error message is sometimes the first thing that is looked at and this can just improve overall clarity. What do you think?

Reply:
The bigger concern is that the stack trace coming with exception may be lost. Can you run a test query to check what the failure message and stack track will be like with this change?

Reply:
You are right, it can cause confusion this way

Reply:
you can supply both the message and the underlying exception